packages/cli/src/imperative.ts

```diff
@@ -29,11 +29,20 @@ import { ZosFilesOptions } from "./zosfiles/ZosFiles.options";
 
 const config: IImperativeConfig = {
     productDisplayName: Constants.DISPLAY_NAME,
-    commandModuleGlobs: ["**/*.definition!(.d).*s"],
+    commandModuleGlobs: [
```

Comment:
Curious why this was changed. If it's still possible to have a single wildcard here, that would be more flexible in the future.

Reply:
This changed because the ** evaluates to any directory that has something with the wildcard *.definition!(.d).*s. So every command definition file falls into that category, from the overall group command definition all the way down to the individual command definitions. I don't know exactly why this behavior occurred, but it caused major problems.